ruby-challenges.rb

```diff
@@ -3,32 +3,66 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
+def even_odd(number)
+    if number % 2 === 0
+        return "#{number} is even"
+    else
+        return "#{number} is odd"
+    end
+end
+
+repostsOne = 7
 # Expected output: '7 is odd'
-reposts2 = 42
+repostsTwo = 42
 # Expected output: '42 is even'
-reposts3 = 221
+repostsThree = 221
 # Expected output: '221 is odd'
 
 
-
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
-beatles_album1 = 'Rubber Soul'
+def remove_vowel(string)
+    if string.is_a? String
+        string.delete "aeiou"
+        else
+            "'#{string}' is not a string value."
+        end
+    end
+    
+    p remove_vowel(beatles_albumOne) # output: "Rbbr Sl"
+    p remove_vowel(beatles_albumTwo) # output: "Sgt Pppr"
+    p remove_vowel(beatles_albumThree) # output: "bby Rd"
+    p remove_vowel(5) # output: "'5' is not a string value."
+    
```

Comment:
I see you're trying to handle edge cases, nice thought! However, the code is not testing for capital letter vowels, so if you were to plug in beatles_albumThree into your method, it would return "Abby Rd" instead of "bby Rd". Also, make sure your indentation is uniform!